packages/vue/src/components/molecules/SfToast/SfToast.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <SfToastContainer :class="containerPositionClass">
+    <transition name="sf-fade" appear>
```

Comment:
what do you think about adding a possibility to change transition too? :)

Reply:
agree :)